pkg/osbuild/mkfs_stages_test.go

```diff
@@ -220,6 +220,88 @@ func TestGenFsStagesBtrfs(t *testing.T) {
 	}, stages)
 }
 
+func TestGenMkfsStagesLVM(t *testing.T) {
```

Comment:
It should probably be called `TestGenFsStagesLVM`. The commit message should be amended as well.

Reply:
ah yes, rebase shenanigans.

Reply:
Or more like "cherry-pick shenanigans".